Formula/webdis.rb

```diff
@@ -0,0 +1,37 @@
+class Webdis < Formula
+  desc "Redis HTTP interface with JSON output"
+  homepage "http://webd.is/"
+  url "https://github.com/nicolasff/webdis/archive/0.1.2.tar.gz"
+  sha256 "8e46093af006e35354f6b3d58a70e3825cd0c074893be318f1858eddbe1cda86"
+
+  depends_on "libevent"
+
+  def install
+    system "make"
+    bin.install "webdis"
+
+    inreplace "webdis.prod.json" do |s|
+      s.gsub! "/var/log/webdis.log", "#{var}/log/webdis.log"
```

Comment:
@DomT4, I'm actually glad you noticed it. I was venting, but my frustration with what happened was very, very real.

I understand that maintainers are not paid to do this, and that they have other commitments, I've done this before myself. As a contributor I too am trying to help the project (I actually have webdis installed and running and don't really need the formula myself — but I thought it'd be neat to provide it for others!).

I've read and followed all the instructions on contributing — wasn't necessarily easy, but I appreciate the thoroughness they provided — and did my best to follow them (all the basics? check, tests? check, better than `--help` tests? check, make sure it passes on the CI? check). Then I was faced with incomplete reviews, no mention of what should I expect further (one reviewer? free-for-all review for a week? do I need to dedicate further time to following up on this?), and reviews based on undocumented practices. At the point where I asked for a second opinion, I was one step away from closing this PR and just dropping the idea of contributing to Homebrew. Please tell me you think "there's no documentation for this practice — just stop arguing and do it" as a review is fair?

Please let me know if you think I'm wrong.